battleship.py

```diff
@@ -172,16 +176,22 @@ def isVertical(ship):
 Returns: bool
 '''
 def isHorizontal(ship):
-    return
-
+    if ship[0][0]==ship[1][0]==ship[2][0]:
+        ship.sort()
+        for col in ship:
+            if ship[0][1]+1==ship[1][1]==ship[2][1]-1:
+                return True
+    return False
 
 '''
 getClickedCell(data, event)
 Parameters: dict mapping strs to values ; mouse event object
 Returns: list of ints
 '''
 def getClickedCell(data, event):
-    return
+    x_coordinate=int(event.x/data["cellSize"])
+    y_coordinate=int(event.y/data["cellSize"])
+    return[y_coordinate,x_coordinate]

```

Comment:
@priyusha99 
Good job. Try checking with the '//' operator in python and figure what does this operation will perform.